src/entities/RfqmJobEntity.ts

```diff
@@ -59,7 +59,7 @@ enum RfqmJobStatus {
  * @deprecated
  * Kept here to ensure existing migrations type check
  */
-export enum RfqmOrderTypes {
+enum RfqmOrderTypes {
```

Comment:
I think you can delete `migrations` and this too

Reply:
looks like there is a bunch of related files. I will delete them in another PR.